lms/static/scripts/frontend_apps/components/FilePickerApp.js

```diff
@@ -63,15 +78,13 @@ function contentDescription(content) {
 export default function FilePickerApp({ onSubmit }) {
   const submitButton = /** @type {{ current: HTMLInputElement }} */ (useRef());
   const {
-    filePicker: {
-      formAction,
-      formFields,
-      canvas: { groupsEnabled: enableGroupConfig, ltiLaunchUrl },
-    },
+    filePicker: { formAction, formFields, ltiLaunchUrl, blackboard, canvas },
   } = useContext(Config);
 
   const [content, setContent] = useState(/** @type {Content|null} */ (null));
 
+  const enableGroupConfig = blackboard?.groupsEnabled || canvas?.groupsEnabled;
```

Comment:
Note here that the frontend really does not care whether we're in a `blackboard` or `canvas` context, but this requires the backend to be rigorous about not ever populating both of these.